client/onboarding/steps/embedded-onboarding.tsx

```diff
@@ -127,6 +127,64 @@ const EmbeddedOnboarding: React.FC = () => {
 			const stripeInstance = loadConnectAndInitialize( {
 				publishableKey: publishableKey,
 				fetchClientSecret: clientSecret, // Pass the function returning the Promise<string>
+				appearance: {
```

Comment:
@oaratovskyi WDYT about moving the styling config into a separate file to keep it separate to our logic. We could do something similar to the `strings.tsx` files we use in our code ([example](https://github.com/Automattic/woocommerce-payments/blob/8f6b267928fb12bee30cb0ba6d1773e9c24b4ff2/client/overview/strings.tsx)) and have a file like `config.tsx` containing the Stripe configuration. If you think it's a good suggestion, I'm happy to add it - just wanted to check with you first!

Reply:
yes, I fully agree!

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/9251/commits/3aab1da52eabbe7690028eca72ff857889452fe7